src-java/base-topology/base-messaging/src/main/java/org/openkilda/messaging/info/switches/LogicalPortsValidationEntry.java

```diff
@@ -0,0 +1,34 @@
+/* Copyright 2021 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.messaging.info.switches;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.Builder;
+import lombok.Data;
+
+import java.io.Serializable;
+import java.util.List;
+
+@Data
+@Builder
+@JsonNaming(value = SnakeCaseStrategy.class)
+public class LogicalPortsValidationEntry implements Serializable {
+    private List<LogicalPortInfoEntry> missing;
```

Comment:
It is terrible. We need to make a concept for handy validation response. :(((

Reply:
What do you mean?